### PR TITLE
chore: migrate monorepo to jest 28

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

- we use jest 27

## New Behavior

- we now follow CDA 100% regarding jest support 🫡
- jest 28 announcement  https://jestjs.io/blog/2022/04/25/jest-28
- migration notes https://jestjs.io/docs/28.x/upgrading-to-jest28

- some v8 packages `tsconfig.json`  now have `ES2015.Collection` as part of their `lib`. This is necessary because we don't use solution configs besides v9 and jest 28 types contain ES2015++ types. Without this build will fail
- react/utilities/test-utilities - refactor to properly expose public API and follow  standards for imports. This issue was exposed thanks to jest 28 and export maps support 🙌

## Related Issue(s)

- Fixes partially https://github.com/microsoft/fluentui/issues/25488

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1dc9d3f16b58efb2f37e930f71b6b140953cd1ea'>1dc9d3f16b58efb2f37e930f71b6b140953cd1ea</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26958&commitId=849279372afc1c25560d89f7c70206b619fa7472&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Avatar-mount_base.html">1582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Avatar-mount.html">1588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Button-mount_base.html">1137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Button-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Field-mount_base.html">2435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Field-mount.html">2440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProvider-mount_base.html">1967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProvider-mount.html">1971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-mount_base.html">765</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-mount.html">753</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">717</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">755</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">762</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/InfoButton-mount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/InfoButton-mount.html">698</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/MakeStyles-mount_base.html">2296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/MakeStyles-mount.html">2274</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Persona-mount_base.html">3626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/Persona-mount.html">3569</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/SpinButton-mount_base.html">2979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test-react-components/SpinButton-mount.html">2997</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/BaseButton-mount_base.html">1559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/BaseButton-mount.html">1560</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Breadcrumb-mount_base.html">3663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Breadcrumb-mount.html">3723</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Checkbox-mount_base.html">3497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Checkbox-mount.html">3457</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/CheckboxBase-mount_base.html">3001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/CheckboxBase-mount.html">3061</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ChoiceGroup-mount_base.html">5935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ChoiceGroup-mount.html">5897</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ComboBox-mount_base.html">1587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ComboBox-mount.html">1597</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/CommandBar-mount_base.html">12027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/CommandBar-mount.html">12022</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ContextualMenu-mount_base.html">18249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ContextualMenu-mount.html">18397</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DefaultButton-mount_base.html">1820</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DefaultButton-mount.html">1845</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRow-mount_base.html">4498</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRow-mount.html">4570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRowFast-mount_base.html">4487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRowFast-mount.html">4539</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRowNoStyles-mount_base.html">4348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DetailsRowNoStyles-mount.html">4334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Dialog-mount_base.html">3894</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Dialog-mount.html">3957</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DocumentCardTitle-mount_base.html">730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/DocumentCardTitle-mount.html">727</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Dropdown-mount_base.html">4263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Dropdown-mount.html">4262</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/FocusTrapZone-mount_base.html">2516</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/FocusTrapZone-mount.html">2603</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/FocusZone-mount_base.html">2471</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/FocusZone-mount.html">2556</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-mount_base.html">69259</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-mount.html">80401</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-virtual-rerender_base.html">32173</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-virtual-rerender.html">32749</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">111988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedList-virtual-rerender-with-unmount.html">109785</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-mount.html">713</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-virtual-rerender_base.html">679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-virtual-rerender.html">662</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">691</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">695</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/IconButton-mount_base.html">2558</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/IconButton-mount.html">2549</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Label-mount_base.html">924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Label-mount.html">927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Layer-mount_base.html">5511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Layer-mount.html">5584</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Link-mount_base.html">1038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Link-mount.html">1076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/MenuButton-mount_base.html">2186</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/MenuButton-mount.html">2241</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/MessageBar-mount_base.html">2989</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/MessageBar-mount.html">2940</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Nav-mount_base.html">4240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Nav-mount.html">4277</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/OverflowSet-mount_base.html">1735</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/OverflowSet-mount.html">1699</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Panel-mount_base.html">3201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Panel-mount.html">3185</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Persona-mount_base.html">1730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Persona-mount.html">1708</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Pivot-mount_base.html">2121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Pivot-mount.html">2045</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/PrimaryButton-mount_base.html">2032</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/PrimaryButton-mount.html">1996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Rating-mount_base.html">9429</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Rating-mount.html">9372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SearchBox-mount_base.html">2016</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SearchBox-mount.html">2020</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Shimmer-mount_base.html">3709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Shimmer-mount.html">3773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Slider-mount_base.html">2685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Slider-mount.html">2733</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SpinButton-mount_base.html">6076</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SpinButton-mount.html">6040</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Spinner-mount_base.html">996</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Spinner-mount.html">1006</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SplitButton-mount_base.html">4063</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SplitButton-mount.html">4121</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Stack-mount_base.html">1079</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Stack-mount.html">1054</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/StackWithIntrinsicChildren-mount_base.html">3192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/StackWithIntrinsicChildren-mount.html">3152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/StackWithTextChildren-mount_base.html">6558</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/StackWithTextChildren-mount.html">6531</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SwatchColorPicker-mount_base.html">13461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/SwatchColorPicker-mount.html">13473</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TagPicker-mount_base.html">3519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TagPicker-mount.html">3401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TeachingBubble-mount_base.html">110021</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TeachingBubble-mount.html">109680</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Text-mount_base.html">991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Text-mount.html">998</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TextField-mount_base.html">2105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/TextField-mount.html">2101</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-mount_base.html">1922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-mount.html">1935</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-virtual-rerender_base.html">1356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-virtual-rerender.html">1336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2702</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Toggle-mount_base.html">1424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/Toggle-mount.html">1461</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/buttonNative-mount_base.html">652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26958/perf-test/buttonNative-mount.html">676</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "type": "patch",
+  "comment": "chore: bump jest types for template",
```
Comment (khmakoto) [inline change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json:3]:
```suggestion
  "comment": "chore: bump jest types for template to v28",
```

**Review by khmakoto — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

**Review by TristanWatanabe — APPROVED** [review]

File: change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26958&commitId=5170a7294f2f62a68492c3c7748f19f786d875a2&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json:1]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26958&commitId=d29a283f7a24ee146f074926f979a8162d63e30f&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Box.Box Example.chromium.png|12| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Review by ling1726 — APPROVED** [review]

**Pull request merged**